service/src/test/java/uk/gov/hmcts/reform/fpl/service/LocalAuthorityUserServiceTest.java

```diff
@@ -30,6 +30,10 @@
     @Mock
     private CaseRoleService caseRoleService;
 
+    @Mock
+    private FeatureToggleService featureToggleService;
```

Comment:
what about the tests? There should be one for toggle on

Reply:
sorted